runtime/compiler/env/SharedCache.hpp

```diff
@@ -80,6 +80,8 @@ class TR_SharedCache
    // and also useful to represent "class chain not present" in places where a class chain offset is optional.
    static const uintptr_t INVALID_CLASS_CHAIN_OFFSET = 0;
 
+   static const uintptr_t INVALID_ROM_CLASS_OFFSET = 0;
```

Comment:
Is this true? Are we guaranteed that a romclass can never be stored at offset 0?

Reply:
Yes, it should be `= 1`.